src/Compatibility/Core/src/Android/Renderers/ListViewAdapter.cs

```diff
@@ -612,6 +612,10 @@ void OnDataChanged()
 				// we need to reset the ListView's adapter to reflect the changes on page B
 				// If there header and footer are present at the reset time of the adapter
 				// they will be DOUBLE added to the ViewGround (the ListView) causing indexes to be off by one. 
+				
+				if (_realListView.IsDisposed())
```

Comment:
Avoid reset ListView adapter in TabbedPage if is already disposed
https://github.com/xamarin/Xamarin.Forms/pull/14635